src/workloads/query/TimeSeriesLastpoint.yml

```diff
@@ -53,369 +54,330 @@ LastpointQueryMetaAscTimeAsc: &LastpointQueryMetaAscTimeAsc
+
+  # QuiesceActor will run in phase 21.
 
   # Lastpoint query with a sort and group with meta subfield ascending and time ascending, but no index.
   - Name: RunLastpointQueryWithMetaSubfieldAscendingTimeAscendingNoIndex
+    Type: CrudActor
+    Database: *db
+    Phases:
+      OnlyActiveInPhases:
+        Active: [22]
+        NopInPhasesUpTo: *MaxPhases
+        PhaseConfig:
+          *LastpointQueryMetaAscTimeAsc
+
+  - Name: CreateIndexForMetaAscTimeAsc
     Type: RunCommand
     Database: *db
     Phases:
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *LastpointQueryMetaAscTimeAsc
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
+      OnlyActiveInPhases:
+        Active: [23]
+        NopInPhasesUpTo: *MaxPhases
+        PhaseConfig:
+          LoadConfig:
+            Path: *phasePath
+            Key: CreateIndex
+            Parameters:
+              Collection: *coll
+              IndexName: *metaDescTimeDesc   # We need to flip the direction for the lastpoint query
+              IndexPattern: *sortPatternMetaDescTimeDesc
+
+  # QuiesceActor will run in phase 24.
 
   # Lastpoint query with a compound index with meta subfield ascending and time ascending.
   - Name: RunLastpointQueryWithMetaSubfieldAscendingTimeAscending
+    Type: CrudActor
+    Database: *db
+    Phases:
+      OnlyActiveInPhases:
+        Active: [25]
+        NopInPhasesUpTo: *MaxPhases
+        PhaseConfig:
+          *LastpointQueryMetaAscTimeAscRepeat100
+
+  - Name: DropIndexForMetaAscTimeAsc
     Type: RunCommand
     Database: *db
     Phases:
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - LoadConfig:
-          Path: *phasePath
-          Key: CreateIndex
-          Parameters:
-            Collection: *coll
-            IndexName: &metaAscTimeAsc "MetaSubfieldAscendingTimeAscending"
-            IndexPattern: *sortPatternMetaAscTimeAsc
-      - *Nop
-      - *LastpointQueryMetaAscTimeAsc
-      - LoadConfig:
-          Path: *phasePath
-          Key: DropIndex
-          Parameters:
-            Collection: *coll
-            IndexName: *metaAscTimeAsc
+      OnlyActiveInPhases:
+        Active: [26]
+        NopInPhasesUpTo: *MaxPhases
+        PhaseConfig:
+          LoadConfig:
+            Path: *phasePath
+            Key: DropIndex
+            Parameters:
+              Collection: *coll
+              IndexName: *metaDescTimeDesc
 
   - LoadConfig:
       Path: *phasePath
       Key: QuiesceActor
       Parameters:
-        MaxPhases: 23
+        MaxPhases: *MaxPhases
```

Comment:
nit: It might be clearer if the active phases for the quiesce actor are defined here rather than inheriting the defaults. 
That way all the phase info is in this file, and it lines up with what is done for TimeSeriesLastpointHighCardinality.yml

Reply:
This is done in #1219.